notebooks/post_DL3_analysis.ipynb

```diff
@@ -1,36 +1,58 @@
 {
```

Comment:
<p>At this points, all the events can be stacked in a single DataSet, that sometimes is easier and faster to handle.</p><p> </p><p><code>stacked_dataset = Datasets(datasets).stack_reduce()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>